### DISCUSSION TITLE
example of batch transaction

### DISCUSSION THREAD

**Discussion by angieyth** [aptos-labs/aptos-developer-discussions#260] [Questions]
### Discord user ID

StopBan05

### Describe your question in detail.

hi, who can give me example how i can create batch transfer transaction?

### What have you tried or looked at?  Or how can we reproduce the error?

N/A

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by angieyth** [comment]
you chan check the example here https://github.com/aptos-labs/aptos-ts-sdk/blob/cd4c5b4490d0f9b170946b79e9080f0a475af4e3/examples/typescript/batch_mint.ts#L36

**Comment by yue1823** [comment]
aptos_framework::aptos_account
fun batch_transfer(source: &signer, recipients: vector<address>, amounts: vector<u64>)

this one you should put your &signer  in first one 
second put a vector of  address 
third put vector of u64 

if you use aptos explorer search 0x1 -> aptos_account -> batch_transfer
you will see three  input 
1.`signer` (just connect your wallet :) )
2. `address1,address2`,.........  (example like this )
3. `amount1,amount2`,..........
